advanced/editor-control-identifiers.md

```diff
@@ -100,9 +105,13 @@ tinyMCE.activeEditor.ui.registry.getAll().buttons
 | cancel               | [save]({{ site.baseurl }}/plugins/save/)                                                                                           | Cancels/Resets the editor contents to it's initial state.                             |
 | save                 | [save]({{ site.baseurl }}/plugins/save/)                                                                                           | Saves the current editor contents to a form or ajax call.                             |
 | searchreplace        | [searchreplace]({{ site.baseurl }}/plugins/searchreplace/)                                                                         | Searches and/or Replaces contents within the editor.                                  |
-| spellchecker         | [spellchecker]({{ site.baseurl }}/plugins/spellchecker/)                                                             | Spellchecks the current editor contents.                                              |
+| spellchecker         | [spellchecker]({{ site.baseurl }}/plugins/spellchecker/)                                                                           | Spellchecks the current editor contents.                                              |
 | table                | [table]({{ site.baseurl }}/plugins/table/)                                                                                         | Creates/Edits table elements.                                                         |
 | tablecellprops       | [table]({{ site.baseurl }}/plugins/table/)                                                                                         | Opens the Cell properties dialog.                                                     |
+| tablecopycol         | [table]({{ site.baseurl }}/plugins/table/)                                                                                         | Copies the current column to the clipboard. {{site.requires_5_4v}}                    |
```

Comment:
As mentioned the other day, the 4 new column ones shouldn't be documented yet (icons and commands are fine, but menu/toolbar items aren't).

Reply:
This also seems to duplicate the other PR? https://github.com/tinymce/tinymce-docs/pull/1560

Reply:
#1560 is menu items, this one just adds toolbar buttons

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1563/commits/086c4d881e172e77661ea2d7ddddf5567acaa960

Reply:
Fixed some but not all references in this PR. I've left a new comment for that.